docs/stacks-guide-aws.md

```diff
@@ -156,693 +175,444 @@ secret/aws-user-creds created
-```
-
-### Route Table
-
-A [Route Table][] specifies rules to direct traffic in a virtual network. We use a
-Route Table to redirect internet traffic from all Subnets to the Internet
-Gateway instance that we created in the previous step.
-
-```bash
-# build route table yaml
-cat > routetable.yaml <<EOF
-apiVersion: network.aws.crossplane.io/v1alpha2
-kind: RouteTable
-metadata:
-  name: ${CONFIG_NAME}-routetable
-  namespace: ${INFRA_NAMESPACE}
-spec:
-  vpcId: ${VPC_ID}
-  routes:
-    - destinationCidrBlock: 0.0.0.0/0
-      gatewayId: ${IG_ID}
-  associations:
-    - subnetId: ${SUBNET1_ID}
-    - subnetId: ${SUBNET2_ID}
-    - subnetId: ${SUBNET3_ID}
-  reclaimPolicy: Delete
-  providerRef:
-    name: aws-provider
-    namespace: ${INFRA_NAMESPACE}
-EOF
-
-# create a routetable object in Crossplane, and wait until the corresponding
-# Route Table in AWS is created and Ready to use
-apply_and_wait_until_ready "routetable.yaml"
-```
-
-Sample output:
-
-```bash
-routetable.network.aws.crossplane.io/aws-network-config-routetable created
-routetable.network.aws.crossplane.io/aws-network-config-routetable condition met
-```
+## Set Up Network Configuration
 
-### Cluster Security Group
+In this section we build a simple AWS network configuration, by creating
+corresponding Crossplane managed resources. These resources are cluster scoped,
+so don't belong to a specific namespace. This network configuration enables
+resources in the WordPress stack to communicate securely. In this guide, we will use
+the [sample AWS network configuration][] in the Crossplane repository. You can read
+more [here][crossplane-gcp-networking-docs] about network secure connectivity
+configurations in Crossplane.
 
-A [Security Group][] is created so that later we can assign it to the EKS cluster. This
-security group allows the cluster to communicate with the worker nodes.
+### TL;DR
 
-```bash
-# build the cluster security group yaml
-cat > cluster_sg.yaml <<EOF
-apiVersion: network.aws.crossplane.io/v1alpha2
-kind: SecurityGroup
-metadata:
-  name: ${CONFIG_NAME}-cluster-sg
-  namespace: ${INFRA_NAMESPACE}
-spec:
-  vpcId: ${VPC_ID}
-  groupName: ${CONFIG_NAME}-cluster-sg
-  description: Cluster communication with worker nodes
-  reclaimPolicy: Delete
-  providerRef:
-    name: aws-provider
-    namespace: ${INFRA_NAMESPACE}
-EOF
-
-# create a cluster security group object in Crossplane, and wait until the corresponding
-# Security Group in AWS is created and Ready to use
-apply_and_wait_until_ready "cluster_sg.yaml"
-```
-
-Sample output:
+Apply the sample network configuration resources:
 
 ```bash
-securitygroup.network.aws.crossplane.io/aws-network-config-cluster-sg created
-securitygroup.network.aws.crossplane.io/aws-network-config-cluster-sg condition met
+kubectl apply -k github.com/crossplaneio/crossplane//cluster/examples/workloads/kubernetes/wordpress/aws/network-config?ref=v0.4.0
```

Comment:
remove extra `/`

Reply:
like the use of ref=v0.4.0 but should we use master here and then snap during the release process or use an env variable?